insilico_panels/KK/MAD_och_MAD-liknande_defekter.v1.0/MAD_och_MAD-liknande_defekter.v1.0

```diff
@@ -0,0 +1,10 @@
+EFTA
+EFTB
+ETFDH
+SLC52A1
+SLC52A2
+SLC52A3
+FLAD1
+RFK
+SLC25A32
+
```

Comment:
Perhaps it is best to the empty line in the end away! Looks tidier and some programs can be stupid ;) (This applies to the other gene lists too.)

Reply:
Yes, I will create bed files in the next step :) And yo´re right, I will tidy it up!